docs/database/README.md

```diff
@@ -76,6 +76,8 @@ db.exec "insert into contacts values (?, ?)", "John", 30
 # Postgres
 db.exec "insert into contacts values ($1, $2)", "Sarah", 33
 ```
+Query parameters are accomplished typically using PreparedStatements under the hood,
+though this can be disabled via a connection string option `prepared_statements=false`.
```

Comment:
Not sure if that statement is correct. I don't think query parameters are technically tied to prepared statements.

Reply:
Also in a147195